RxExample/RxExample/Examples/ImagePicker/ImagePickerController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  ImagePickerController.swift
+//  RxExample
+//
+//  Created by Segii Shulga on 1/5/16.
+//  Copyright © 2016 Krunoslav Zaher. All rights reserved.
+//
+
+import Foundation
+import UIKit
+#if !RX_NO_MODULE
+   import RxSwift
+   import RxCocoa
+#endif
+
+struct ImagePickerConfiguration {
+   let soruceType:UIImagePickerControllerSourceType
+   let allowsEditing:Bool
+}
+
+class ImagePickerController: ViewController {
+
+   @IBOutlet var imageView: UIImageView!
+   @IBOutlet var cameraButton: UIButton!
+   @IBOutlet var galleryButton: UIButton!
+   @IBOutlet var cropButton: UIButton!
```

Comment:
@sergdort I'm not doing some work on this code, and it seems that the code you've written there is `tab=3 spaces` :)

This is kind of funny situation, just out of curiosity, do you have tab set to 3 spaces? Because it seems that spaces are divisible by 3 :)